### DISCUSSION TITLE
Need help to migrate RabbitMQ from classic queue to quorum queue without cluster downtime.

### DISCUSSION THREAD

**Discussion by daviddongfanzhong** [rabbitmq/rabbitmq-server#11136] [Questions]
Need help to migrate RabbitMQ from classic queue to quorum queue without cluster downtime.

We have a 3 node cluster production environment. They are about 1000 classic queues running. We recently upgraded RabbitMQ to 3.13.0 from 3.11.8. We want to migrate the classic queues to quorum queues without downtime. Anyone has idea on how to do this migration? We need your help, thanks. -- David

**Comment by lukebakken** [comment]
https://www.rabbitmq.com/docs/blue-green-upgrade
